java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -348,25 +348,25 @@ public interface StringBaseCommands {
 
     /**
      * Returns the indices and length of the longest common subsequence between strings stored at
-     * <code>key1</code> and <code>
-     * key2</code>.
+     * <code>key1</code> and <code>key2</code>.
      *
      * @since Redis 7.0 and above.
      * @apiNote When in cluster mode, <code>key1</code> and <code>key2</code> must map to the same
      *     hash slot.
      * @see <a href="https://valkey.io/commands/lcs/">valkey.io</a> for details.
      * @param key1 The key that stores the first string.
      * @param key2 The key that stores the second string.
-     * @return A <code>HashMap<String, Object</code> containing the indices of longest common
-     *     subsequence between the 2 strings. The <code>Object</code> mapped to the <code>"matches"
-     *     </code> String contains a two-dimensional Long array that stores the pair of start and end
-     *     indices of the first and second Strings that match. An empty <code>Object</code> in the
-     *     <code>HashMap</code> is returned if the keys do not exist or have no common subsequences.
+     * @return A <code>Map&lt;String, Object&gt;</code> containing the indices of the longest common
```

Comment:
Just wanted to double check that using `&lt;` instead of `<` was intentional? I'm assuming that's how you need to represent `<` inside a code block?

Reply:
You can use just a `Map`

Reply:
Yes, that how we can show `<` in html.